README.md

```diff
@@ -32,17 +32,17 @@ Soft Forks & protocol overview|Fork categorization (soft, hard, evil, etc.)|[Bet
 Soft Forks & protocol overview|BIP91|[BIP 91](https://github.com/bitcoin/bips/blob/master/bip-0091.mediawiki)|
 Security Models|Overview|[Security models seminar video](https://youtu.be/6gGcS4N5Rg4), [Overview of Security Concerns](https://diyhpl.us/wiki/transcripts/scalingbitcoin/overview-of-security-concerns/), [Danger! Bitcoin Threat Models](https://www.youtube.com/watch?v=ddN58UqKPx0#t=750), [Bitcoin’s Security Model: A Deep Dive](https://www.coindesk.com/bitcoins-security-model-deep-dive), [Weaknesses](https://en.bitcoin.it/wiki/Weaknesses), [Speed-Security Tradeoffs in Blockchain Protocols](https://pdfs.semanticscholar.org/ac1a/918fc933b767d34574ec2cc6a33b4223dc1a.pdf), [The Onion Model of Blockchain Security](https://insights.deribit.com/market-research/the-onion-model-of-blockchain-security-part-1/)|
 Security Models|Checkpoints, assumevalid, minimumchainwork|[Bitcoin's Diversity of Use-Cases and Security Models](https://bluematt.bitcoin.ninja/2017/02/28/bitcoin-trustlessness/), [Dave Harding Tweet on assumed valid blocks and minimum chainwork](https://twitter.com/hrdng/status/869206705548271616)|
 Security Models|Defining SPV and lightclients|[Light Clients](https://gist.github.com/ariard/1034cd7624805d53334e80d4712fb8ee)|
 Security Models|BIP 37 (bloom filters)|[BIP 37](https://github.com/bitcoin/bips/blob/master/bip-0037.mediawiki), [On the Privacy Provisions of Bloom Filters in Lightweight Bitcoin Clients](https://eprint.iacr.org/2014/763.pdf)|
-Security Models|Neutrino |[BIP 157](https://github.com/bitcoin/bips/blob/master/bip-0157.mediawiki), [Neutrino: The Lighter Side of Lightning](https://blog.lightning.engineering/posts/2018/10/17/neutrino.html), [Index for BIP 157 block filters](https://github.com/bitcoin/bitcoin/pull/14121), [Exploring Neutrino](https://vimeo.com/316626387)|
+Security Models|Neutrino |[BIP 157](https://github.com/bitcoin/bips/blob/master/bip-0157.mediawiki), [Neutrino: The Lighter Side of Lightning](https://blog.lightning.engineering/posts/2018/10/17/neutrino.html), [Index for BIP 157 block filters](https://github.com/bitcoin/bitcoin/pull/14121)|
 Security Models|Committed bloom filters|[Committed bloom filters for improved wallet performance and SPV security](https://lists.linuxfoundation.org/pipermail/bitcoin-dev/2016-May/012636.html)|
 Security Models|Fraud proofs|[Fraud Proofs (2018)](https://diyhpl.us/wiki/transcripts/scalingbitcoin/tokyo-2018/fraud-proofs/), [Fraud Proofs: Improving the ability of SPV clients to detect invalid chains (gist)](https://gist.github.com/justusranvier/451616fa4697b5f25f60), [Bitcoin Wizards Fraud Proof thread](https://people.xiph.org/~greg/bitcoin-wizards-fraud-proof.log.txt), [Improving SPV security with PoW fraud proofs](https://www.mail-archive.com/bitcoin-dev@lists.linuxfoundation.org/msg07913.html)|
```

Comment:
Greg Maxwell's server appears down. We'll have to pull out this too.

Reply:
okay, got it.
So I removed all links related to "https://people.xiph.org"